### PR TITLE
[.NET] Turkish NumbersWithUnit (Currency) support

### PR DESCRIPTION
**Pull request by LionbridgeCS**
Support for Currency in Turkish. 
Passing: 95
Skipped: 2

Explanation: 
"sadece on üç dolar kırk beş sente mal oldu" => Sente is a suffixed version of sent which is ambiguous with the actual currency unit sente. 
"12 Aralık 1994 itibariyle 125 milyon avustralya doları sıfır - kupon eurobond, 50.9375'te fiyatlandı ve hambros bankası şirketi aracılığıyla % 15.06 daha az ücret verdi." => Trukish does not have equivalent for -of-, sıfır is included in the number and very hard to extract.

### CODE REVIEW

**Comment by LionbridgeCS** [comment]
@tellarin, the build fails because of the Java dependency downloading issue. we tried closing and reopening the PR but we still did not get more info. Is there something we can do?

**Comment by LionbridgeCS** [comment]
@tellarin, the build passe now, closing and re opening seems to have done it.

**Comment by LionbridgeCS** [comment]
@tellarin, this PR is ready for review and possible feedback.

File: .NET/Microsoft.Recognizers.Text.DataDrivenTests/NumberWithUnit/TestNumberWithUnit_Turkish.cs [hunk 1]
```diff
@@ -26,5 +26,12 @@ public void TemperatureModel(TestModel testSpec)
         {
             TestNumberWithUnit(testSpec);
         }
+
+        [NetCoreTestDataSource]
+        [TestMethod]
+        public void CurrencyModel(TestModel testSpec)
+        {
+            TestNumberWithUnit(testSpec);
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Text.DataDrivenTests/NumberWithUnit/TestNumberWithUnit_Turkish.cs]: English calls TestCurrency(), why does Turkish calls TestNumberWithUnit() if it should "branch" from English?

**Pull request merged**